app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,19 +27,39 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * 모든 Product를 반환한다.
+     * @return 모든 Product
+     */
```

Comment:
모든 Product 목록이라고 `목록`을 명시해주는 것은 어떨까요?
```suggestion
    /**
     * 모든 Product 목록 반환한다.
     * @return 모든 Product 목록
     */
```